internal/service/ipam/model_ipamsvc_ha_group_host.go

```diff
@@ -0,0 +1,121 @@
+package ipam
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/ipam"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type IpamsvcHAGroupHostModel struct {
+	Address    types.String `tfsdk:"address"`
+	Heartbeats types.List   `tfsdk:"heartbeats"`
+	Host       types.String `tfsdk:"host"`
+	Port       types.Int64  `tfsdk:"port"`
+	Role       types.String `tfsdk:"role"`
+	State      types.String `tfsdk:"state"`
+}
+
+var IpamsvcHAGroupHostAttrTypes = map[string]attr.Type{
+	"address":    types.StringType,
+	"heartbeats": types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcHAGroupHeartbeatsAttrTypes}},
+	"host":       types.StringType,
+	"port":       types.Int64Type,
+	"role":       types.StringType,
+	"state":      types.StringType,
+}
+
+var IpamsvcHAGroupHostResourceSchemaAttributes = map[string]schema.Attribute{
+	"address": schema.StringAttribute{
+		Computed:            true,
```

Comment:
This is not a read only field and can be provided by an user. It should have both Optional and Computed as true

Reply:
No, the address has to be a computed field. The value is retrieved by the API from the `hosts` table.

Reply:
You have the option to change that address in the UI.